Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: help install dev-install clean lint format test test-cov run-backend run-frontend run pre-commit-install pre-commit-run
+.PHONY: help install dev-install clean lint format typecheck test test-cov run-backend run-frontend run pre-commit-install pre-commit-run
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)